server/util/aria.js

```diff
@@ -8,7 +8,14 @@ const evaluateAtNameKey = atName => {
 };
 
 const testWithNoIds = test => ({
```

Comment:
It seems like it might be time to update this variable name since it's now responsible for filtering a lot of information which is not really IDs.

Reply:
Good catch! Done in https://github.com/w3c/aria-at-app/pull/880/commits/eeb2fb8f1e11176d3c4e27a40c05d2e2f1ce1ed4